app/javascript/packs/stylesheets/layout.scss

```diff
@@ -18,3 +18,9 @@
   font-style: italic;
   float: right;
 }
+
+.jursdiction-user-box {
+  float: right;
+  background-color: #e9ecef;
+  padding: 3px 8px;
+}
```

Comment:
small thing, but I think we usually leave an extra line at the end of every file?

Reply:
done